test/entrypoint.sh

```diff
@@ -30,6 +30,13 @@ MYSQL_CONTAINER=1 $DIR/create_db.sh
 # Set up rabbitmq exchange
 rabbitmq-setup -server amqp://boulder-rabbitmq
 
+# Make sure SoftHSM container is up and using the correct key
+wait_tcp_port boulder-hsm 5657
+if [ ! -f test/test-ca.key.der ]; then
+    openssl rsa -in test/test-ca.key -inform pem -out test/test-ca.key.der -outform der
+fi
+PKCS11_PROXY_SOCKET="tcp://boulder-hsm:5657" pkcs11-tool --module=/usr/local/lib/libpkcs11-proxy.so --write-object test/test-ca.key.der --type privkey --label key_label --pin 5678 --login --so-pin 1234
```

Comment:
I don't think this is possible without adding another image that copies in the key (or doing something a bit funky with volumes). Since we are already calling `create_db.sh` in `entrypoint.sh` this change seems in line with how the checks currently work.

I could add a test that only tries to write the key if the key currently in SoftHSM doesn't match the one from source in a similar manner to `create_db.sh` but I think that'd actually take longer than just overwriting the key each time.